lib/pg-log/index.ts

```
@@ -1,10 +1,26 @@
 import IsLogger from '../is_logger';
+import createPgliteClient from './pglite-adapter';
 
-const pool: Pool = new Pool({
-  max: 3,
-  connectionTimeoutMillis: 2000,
-});
+export type DbClient = {
+  query(...args: [string, any[]?]): Promise<{ rows: any[]; rowCount: number | null; command?: string }>;
+};
+
+let _pool: DbClient | null = null;
```

ESLint

[error] Unexpected dangling '_' in '_pool'.

(no-underscore-dangle)